### PR TITLE
WIP - Module 3 - Observability

### PR DESCRIPTION
**Pull request by mrhoads**
This is still a WIP but I'm submitting a PR for my initial draft for observability.  I need to add additional screenshots on Azure Monitor / Container Insights and briefly describe that more but wanted to capture work already done for early review.

### CODE REVIEW

File: labguide/IPD2024.code-workspace
Comment (dkirby-ms) [inline labguide/IPD2024.code-workspace:1]: whats this for?

**Review by dkirby-ms — APPROVED** [review]
It looks fine. If we can find a way to also include an OT dashboard that Zaid built to show the data from the simulator that could be cool. 

Do you think it makes sense to include any pre-staged components on the cluster to avoid user having to do it?

File: labguide/IPD2024.code-workspace
Comment (mrhoads) [reply labguide/IPD2024.code-workspace:1]: I didn't mean to commit this.  It's the vscode workspace I've been using.  I'll remove it

**Comment by mrhoads** [comment]
> It looks fine. If we can find a way to also include an OT dashboard that Zaid built to show the data from the simulator that could be cool.
> 
> Do you think it makes sense to include any pre-staged components on the cluster to avoid user having to do it?

I'll work on including the simulator and the OT dashboard.  If we need to condense this then yeah, we can already have the kube-prom-stack already deployed

**Pull request merged**